bindings/python/pymanifold.cpp

```diff
@@ -95,7 +95,7 @@ PYBIND11_MODULE(pymanifold, m) {
       .def(py::self ^ py::self, "Boolean intersection.")
       .def(
           "transform",
-          [](Manifold self, py::array_t<float> &mat) {
+          [](Manifold &self, py::array_t<float> &mat) {
```

Comment:
Python is not my forte - what does this change?

Reply:
I *think* (cannot find this in the documentation) pybind will not perform a copy when the argument is a reference. I don't think it makes a difference for us as our Manifold objects are copied lazily. Nonetheless this should not hurt.

Reply:
Ah ok, I didn't know that there was a lazy copy mechanism under the hood. But like you say, it's better to send the object as a reference anyway.